src/main/java/frc/robot/subsystems/Balancer.java

```diff
@@ -0,0 +1,73 @@
+package frc.robot.subsystems;
+import com.kauailabs.navx.frc.AHRS;
+import edu.wpi.first.wpilibj.SPI;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Balancer extends SubsystemBase {
+
+    public int phase = 1;
```

Comment:
usually it's clearer to use java ENUMS to represent states, instead of ints, for readability